internal/buffer/save.go

```diff
@@ -24,6 +26,43 @@ import (
 // because hashing is too slow
 const LargeFileThreshold = 50000
 
+type saveRequest struct {
+	*Buffer
+	path      string
+	withSudo  bool
+	errorChan chan error
+}
+
+var saveRequestChan chan saveRequest
+var backupRequestChan chan *Buffer
+
+func init() {
+	saveRequestChan = make(chan saveRequest, 10)
+	backupRequestChan = make(chan *Buffer, 10)
+
+	go func() {
+		var elapsed <-chan time.Time
+		duration := backupSeconds * float64(time.Second)
+		backupTicker := time.NewTicker(time.Duration(duration))
+		elapsed = backupTicker.C
```

Comment:
nit: I don't see much point in using this `elapsed` variable instead of directly using `backupTicker.C`. And it even makes the code less clear,  since it doesn't tell what exactly elapsed.